site/content/events/2015-perth/index.html

```diff
@@ -4,47 +4,48 @@
 filter:
 - erb
 - markdown
-title: 'Welcome !'
-layout: event
+title: 'Welcome!'
 dirty: true
-platinum: true
 ---
 <% @eventhome = @page.directory.split(File::SEPARATOR)[0..1].join(File::SEPARATOR) %>
 
-
 ## Wednesday, October 14th and Thursday, October 15th
 
 Welcome!  The DevOps Days comes to Perth for the first time. Come be a part and create history with us.
+<div style="text-align:center;">
+   <img alt="DevOpsDays Perth 2015" src="./logo.png"/>
+</div>
 
 <table>
-	<tr>	
-		<td><img border=0 src="logos/dodperth2015.png" alt="DevOps Days Perth"></td>
-		<td><img border=0 width="160" height="240" src="./perth.png"></td>
+  <tr>
+    <td><b>Date</b></td>
+    		<td> <%= render(:partial => "/#{@eventhome}/_event_date_start") %> - <%= render(:partial => "/#{@eventhome}/_event_date_end") %></td>
+  </tr>
+  <tr>
+    <td><b>Location</b></td>
+	<td><a href="./location">Bankwest Place Murray Street</a></td>
+  </tr>
+  <tr>
+    <td><b>Participants</b></td>
+    <td><a href="./registration">Register to attend the event!</a></td>
+  </tr>
+	<tr>
+	  <td><b>Propose a talk</b>!</td>
+	  <td><a href="./propose"> Propose</td>
 	</tr>
-	  <tr>
-		<td><b>Date</b></td>
-		<td> <%= render(:partial => "/#{@eventhome}/_event_date_start") %> - <%= render(:partial => "/#{@eventhome}/_event_date_end") %></td>
-	  </tr>
-	  <tr>
-		<td><b>Location</b></td>
-		<td><a href="./location">Bankwest Place Murray Street</a></td>
-	  </tr>
-		<tr>
-		  <td><b>Propose a talk</b>!</td>
-		  <td><a href="./propose"> Propose</td>
-		</tr>
-		<tr>
-			<td><b>Participants</b></td>
-			<td><a href="./registration">Register to attend the event!</a></td>
-		</tr>
-		<tr>
-			<td><b>Sponsor</b></td>
-			<td><a href="./sponsor">Sponsor the event</a> ! It's a great way to promote your organization.</td>
-		</tr>
-		<tr>
+  <!--tr>
+    <td><b>Program</b></td>
+    <td>Check out the current <a href="/<%= @eventhome %>/program/">program</a>.</td>
+  </tr-->
+	<tr>
+		<td><b>Sponsor</b></td>
+		<td><a href="./sponsor">Sponsor the event</a> ! It's a great way to promote your organization.</td>

```

Comment:
Space between "Sponsor the event" and the punctuation mark. I suspect you copied this from a Francophone template. :wink: